src/main/java/org/broadinstitute/hellbender/engine/ReadWalker.java

```diff
@@ -29,7 +24,7 @@
  * ReadWalker authors must implement the apply() method to process each read, and may optionally implement
  * onTraversalStart() and/or onTraversalSuccess(). See the PrintReadsWithReference walker for an example.
  */
-public abstract class ReadWalker extends GATKTool {
+public abstract class ReadWalker extends Walker {
```

Comment:
Update docs for `ReadWalker.traverse()` to read "You should only override traverse() if you are writing a new walker base class in the engine package that extends this class. It is not meant to be overridden by tools outside of the engine package." instead of the current "Subclasses can override to provide own behavior"

Reply:
Fixed.